apiserver/facades/controller/caasapplicationprovisioner/state.go

```diff
@@ -45,9 +43,7 @@ type CAASApplicationControllerState interface {
 
 type Model interface {
 	UUID() string
```

Comment:
Can we please do one more drive by in this PR. So we still retain the model's `UUID` on model config but we want to blow away these uses where we can.

I think you will find this state method is currently not being used as all uses of reading the models id are coming off of the model config object.

There now exists a model information service (https://github.com/juju/juju/blob/main/domain/model/service/modelservice.go) where you can pull model info like the id. Can we please plumb this in and make all use of reading the model id pull off of this so we can blow model id away from model config after we are done?

Reply:
Above on the `CAASApplicationControllerState` there is a `ModelUUID` func that has no uses.  Can we blow this away in the PR please.